Logger/BaseLogger.cs

```diff
@@ -1,7 +1,8 @@
-﻿namespace Logger
+﻿    namespace Logger
 {
     public abstract class BaseLogger
     {
+        public string className { get; set; }
```

Comment:
Properties should be PascalCase
```suggestion
        public string ClassName { get; set; }
```